packages/kotti-ui/source/kotti-banner/KtBanner.vue

```diff
@@ -1,81 +1,109 @@
 <template>
 	<div :class="bannerClass">
-		<div class="collapse">
-			<div class="glyph">
-				<span v-if="icon">
+		<div class="kt-banner__collapse">
+			<div class="kt-banner__glyph">
+				<slot name="glyph">
 					<i class="yoco" v-text="icon" />
-				</span>
-				<span v-else>
```

Comment:
Technically a breaking change, as it changes the behavior if both the icon and slot are passed. Needs to be documented in the next release (and therefore PR description) and needs to be sanity-checked in yoda.

Reply:
Well, the breaking change will be, that we abolish this slot entirely. It is not used in yoda whatsoever btw